app.py

```diff
@@ -1,6 +1,17 @@
 #!/usr/bin/env python
 
+import psycopg2
+
 from beyond import app
 
+config = {
+    'host': 'postgres',
+    'port': 5432,
+    'user': 'beyond',
+    'password': 'beyond',
+    'dbname': 'beyond'
+}
+
 if __name__ == '__main__':
     app.run(host='0.0.0.0', debug=True)
+    conn = psycopg2.connect(**config)
```

Comment:
What is the purpose of connecting the database here?

Reply:
> What is the purpose of connecting the database here?

Since we added a new container with a DB, and mapped it to the app container, we wish to check that not only the new container as hanged on the air by itself, but we also wish to check whether we can connect to it as well.
Otherwise, this new container addition doesnt make a lot of sense.

Reply:
Is checking whether we can connect to the db container is the purpose of this service?

Reply:
> Is checking whether we can connect to the db container is the purpose of this service?

This is not a test, but just establishing a connection to our DB. So yes, if the DB isnt configured right, then the connection wouldnt work.

Reply:
I got what you are saying. no need for this yet.
Removing.